### PR TITLE
[New] Deploy cPanel with One-Click Apps

### PR DESCRIPTION
**Pull request by leslitagordita**
How to Deploy cPanel with One-Click Apps

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit d5efdaf7f1fab5921ac3f37639df2858223635c0

https://deploy-preview-3205--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/platform/one-click/how-to-deploy-cpanel-with-one-click-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'cPanel & WHM is a Linux based web hosting control panel and platform that helps you create and manage websites, servers, databases and more with a suite of hosting automation and optimization tools.  Deploy a cPanel & WHM instance using Linode''s One-Click Apps.'
```
Comment (andystevensname) [inline docs/platform/one-click/how-to-deploy-cpanel-with-one-click-apps/index.md]: Two commas for both description and og_description: .."databases, and..." and "..and more, with..." Also, what does the double apostrophe in Linode''s do? We could use quotation marks around the entire description if we're worried about escaping that single apostrophe in "Linode's."

File: docs/platform/one-click/how-to-deploy-cpanel-with-one-click-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+modified_by:
+  name: Linode
+title: "How to Deploy cPanel with One-Click Apps"
+h1_title: "Deploy cPanel with One-Click Apps"
+contributor:
+  name: Linode
+external_resources:
+- '[WHM Feature Documentation](https://documentation.cpanel.net/display/78Docs/WHM+Features+List)'
+---
+The [cPanel & WHM](https://cpanel.net/products/) One-Click App streamlines publishing and managing a website on your Linode. cPanel & WHM is a Linux based web hosting control panel and platform that helps you create and manage websites, servers, databases and more with a suite of hosting automation and optimization tools.
```
Comment (andystevensname) [inline docs/platform/one-click/how-to-deploy-cpanel-with-one-click-apps/index.md]: Same comma scenario as above.

File: docs/platform/one-click/how-to-deploy-cpanel-with-one-click-apps/index.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Linode Options
+
+Provide configurations for your Linode server. The table below includes details about each configuration option.
+
+| **Configuration** | **Description** |
+|-------------------|-----------------|
+| **Select an Image** | CentOS 7 is currently the only image supported by the WireGuard One-Click App, and it is pre-selected on the Linode creation page. *Required* |
```
Comment (andystevensname) [inline docs/platform/one-click/how-to-deploy-cpanel-with-one-click-apps/index.md]: Remove mention of WireGuard.

**Pull request merged**